src/core/packages/http/common/src/constants.ts

```diff
@@ -11,6 +11,7 @@
 export const ELASTIC_HTTP_VERSION_HEADER = 'elastic-api-version' as const;
 export const ELASTIC_HTTP_VERSION_QUERY_PARAM = 'apiVersion' as const;
 export const ELASTIC_INTERNAL_ORIGIN_QUERY_PARAM = 'elasticInternalOrigin' as const;
+export const ELASTIC_PUBLIC_FILTER_PATH_QUERY_PARAM = 'filter_path' as const;
```

Comment:
This is a global unique, snake case, query parameter. Risk is that some app is already using it, I searched across the code base and could only see a couple uses of this by Kibana....

Reply:
> Risk is that some app is already using it, I searched across the code base and could only see a couple uses of this by Kibana....

It'll be useful to know where and why they're using `filter_path` and if it's to reduce the response body or some other reason.